### DISCUSSION TITLE
Update to most recent.

### DISCUSSION THREAD

**Discussion by miodzicho** [RobertDWhite/WhiteMatterTech#11] [General]
Hi, first, thank you for great work bringing Graylog in such a simple wrap up.
Two questions, can I add icons on unraid ?
How I can update it to recent versions ? Updating stack does nothing..

**Comment by RobertDWhite** [comment]
Hello @miodzicho 
First, when creating the stack, you should have options to add custom icons. I do not currently have an Unraid installation, unfortunately, or I would screenshot and show you exactly how. However, the feature is there.
Second, to update to recent versions, you'll need to update the section in the docker-compose next to the container repo/name. For example
```
    image: "graylog/graylog:4.2"
```
In the above line, you would change 4.2 to any valid Graylog container tag. You can find the available tags here: https://hub.docker.com/r/graylog/graylog/tags

Be aware, though, that changing the version may have compatibility issues with the rest of the stack. You may have to update other versions of the rest of the stack accordingly. 

Thanks!
